### PR TITLE
Feature/fix availability

### PR DESCRIPTION
**Pull request by tripox**

### CODE REVIEW

File: modules/aleph/includes/aleph.availability.inc [hunk 1]
```diff
@@ -94,21 +93,15 @@ function aleph_availability_holdings(array $provider_ids) {
...
-      // libraries.
-      if (!$item->isReservable()) {
-        $reserved_count++;
-      }
-      else {
-        $available_count++;
-      }
+      // Count how many of the materials are on shelf / available and how
+      // many are reservable.
+      $item->isAvailable() ? $available_count++ : $reserved_count++;
```
Comment (kasperg) [inline modules/aleph/includes/aleph.availability.inc:98]: ⛳️

File: modules/aleph/includes/aleph.availability.inc [hunk 2]
```diff
@@ -94,21 +93,15 @@ function aleph_availability_holdings(array $provider_ids) {
...
 
       // Provide data for each material from the provider ID.
       $data = array(
         'local_id' => $item->getId(),
         'available' => $available_count > 0,
-        'reservable' => $available_count > 0,
-        'show_reservation_button' => $available_count > 0,
+        'reservable' => count($items) > 0,
```
Comment (kasperg) [inline modules/aleph/includes/aleph.availability.inc]: This means that a record will always be reservable if any items exist. That does not seem right. 

How about checking if the number of reservable items is more than 0?

File: modules/aleph/includes/aleph.availability.inc [hunk 3]
```diff
@@ -47,7 +47,6 @@ function aleph_availability_items(array $provider_ids) {
  *   Holdings info.
  */
 function aleph_availability_holdings(array $provider_ids) {
-  $data = array();
```
Comment (kasperg) [inline modules/aleph/includes/aleph.availability.inc:50]: :+1:

File: modules/aleph/includes/aleph.availability.inc (hunk 2 above)
Comment (kasperg) [reply modules/aleph/includes/aleph.availability.inc]: Also: Is it intentional that you remove the `show_reservation_button` entry?

Comment (tripox) [reply modules/aleph/includes/aleph.availability.inc]: `show_reservation_button` doesn't seem to have any functionality at all.
This is determined by `reservable`.

**Review by kasperg — APPROVED** [review]

**Pull request merged**